client/messages/es.json

```
@@ -1,4 +1,388 @@
 {
+  "cQyKR1": "Endpoints",
```

Biome (2.4.13)

[error] The key cQyKR1 was already declared.

(lint/suspicious/noDuplicateObjectKeys)